setup.sh

```diff
@@ -47,6 +48,7 @@ $env/bin/pip install -U setuptools
 echo -e "${info}[*] Installing Python packages into $env...${reset}"
 if [ "$1" == "server" ]; then
   $env/bin/pip install -r $codalabdir/requirements-server.txt
+  $env/bin/pip install -e $workerdir
```

Comment:
Nit: Why define `workerdir` if it's going to be used once?  If put `$codalabdir/worker` here directly, it's clearer that they're both in `$codalabdir`...